misk/src/main/kotlin/misk/environment/EnvironmentModule.kt

```diff
@@ -1,10 +1,26 @@
 package misk.environment
 
 import com.google.inject.AbstractModule
+import misk.logging.getLogger
 
-class EnvironmentModule(private val environment: Environment) : AbstractModule() {
+private val logger = getLogger<EnvironmentModule>()
+
+class EnvironmentModule(val environment: Environment) : AbstractModule() {
     override fun configure() {
         bind(Environment::class.java).toInstance(environment)
     }
 
+    companion object {
+        @JvmStatic
+        fun fromEnvironmentVariable(): EnvironmentModule {
+            val environmentName = System.getenv("ENV")
```

Comment:
Should we call this `ENV` or `ENVIRONMENT` ? I’m partial to `ENVIRONMENT` because I hate having multiple different names for the same thing.

Reply:
Let's go environment. I tried to keep everything as `Environment` instead of `ENV` in the code, but missed it here. Will fix.